### PR TITLE
Restore Laravel fixture app test

### PR DESCRIPTION
**Pull request by bakerkretzmar**
This PR adds back a test using a freshly installed Laravel app. These tests were deleted (mistakenly, I think) in 77efe50bb1ce1bb5fc1dabf57d62563c2468a96d, which updated the fixture app to Laravel 7 but also deleted its tests.

Also: tweaked the formatting of `run-tests.yml` and `phpunit.xml`, and added a Composer script to automate updating the fixture app in the future. The reason that script deletes the fixture app's `composer.lock` is to avoid GitHub security alerts for out of date dependencies.

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ jobs:
         php: ['7.3', '7.4', '8.0']
         os: [ubuntu-latest, macos-latest]
         dependency-version: [prefer-lowest, prefer-stable]
+        phpunit: ['9.*']
         include:
-          - phpunit: '9.*'
           - php: '7.3'
             phpunit: '8.*'
```
Comment (mattstauffer) [inline .github/workflows/run-tests.yml:20]: I know this isn't a change you made in this PR, but is it not weird that we have PHPUnit 8.* on the PHP, and Phpunit 9.* on its own line?

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -24,29 +24,19 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Cache dependencies
-        uses: actions/cache@v2
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
```
Comment (mattstauffer) [inline .github/workflows/run-tests.yml:28]: Do you know what these two look like without a name?

I know the name generates this UI:

image

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ name: Tests
```
Comment (mattstauffer) [inline .github/workflows/run-tests.yml:2]: While we're in here, can we change the name to be "Run Tests"

Comment (mattstauffer) [reply .github/workflows/run-tests.yml:2]: (or "Run tests")

**Review by mattstauffer — COMMENTED** [review]
Good work. a few notes

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (bakerkretzmar) [reply .github/workflows/run-tests.yml:28]: Without a name it uses "Run <action-name>", e.g. `Run actions/cache@v2`

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (bakerkretzmar) [reply .github/workflows/run-tests.yml:20]: `include` is supposed to be specifically for overrides and 'extra' stuff afaik, but PHPUnit is used in every test run and we just need to override the version that one time, so it should have had its own key in the `matrix` all along.

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (bakerkretzmar) [reply .github/workflows/run-tests.yml:28]: The actions that are just scripts show the first line of the script, so e.g. the Composer one would show `Run composer require phpunit/phpunit:${{ matrix.phpunit }} --no-update --no-interaction`

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (mattstauffer) [reply .github/workflows/run-tests.yml:20]: 👍 Thanks!

**Comment by mattstauffer** [comment]
If you think the tests look better with the non-named version, go for it!

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (bakerkretzmar) [reply .github/workflows/run-tests.yml:20]: @mattstauffer turns out I was very wrong about this! `include` is for 'extra' stuff but NOT for overrides. The previous syntax was correct so I've reverted it.

**Pull request merged**